src/build.rs

```diff
@@ -2,7 +2,13 @@ pub async fn build() -> fpm::Result<()> {
     let config = fpm::Config::read().await?;
     tokio::fs::create_dir_all(format!("{}/.build", config.root.as_str()).as_str()).await?;
 
-    for doc in fpm::process_dir(config.root.as_str(), &config, None).await? {
+    for doc in fpm::process_dir(
+        config.root.as_str(),
+        &config,
+        fpm::ignore_paths(vec![".history", ".FTD"]),
```

Comment:
What is `.FTD` folder for?

Also I feel people should not have to send the ignore_paths like that. fpm::process_dir must be smart enough to know what are the files in ftd package. Process dir knows about .gitignore handling, it must have already ignore these two as well I feel.